pkg/apps/controller/util/util.go

```diff
@@ -74,12 +75,6 @@ func DecodeDeploymentConfig(controller metav1.ObjectMetaAccessor) (*appsapi.Depl
 	return config, nil
 }
 
-// DeployerPodSelector returns a label Selector which can be used to find all
```

Comment:
@deads2k you asked about number of remaining internal helpers... these are left overs (used by REST and controllers)... Note that not all if them are duplicated into `pkg/apps/util`, some live sorely to make just controllers happy.

Reply:
> @deads2k you asked about number of remaining internal helpers... these are left overs (used by REST and controllers)... Note that not all if them are duplicated into pkg/apps/util, some live sorely to make just controllers happy.

glad to see it moving in the right direction